app/Services/Users/CleanupService.php

```diff
@@ -57,6 +59,12 @@ protected function removeFollows(): self
         return $this;
     }
 
+    protected function removeFeatureRequests(): self
+    {
+        Feature::where('created_by', $this->user->id)->where('upvote_count', '<', 10)->delete();
```

Comment:
Only delete ideas that are in the "voting" phase. Ideas that have committed to working on, or already delivered, shouldn't disappear :-)